content/well-architected-framework/docs/docs/secure-systems/secure-applications/authenticate-certificates/authenticate-workloads-tls.mdx

```diff
@@ -0,0 +1,82 @@
+
+![Authenticate applications with TLS certificates through Vault Agent](/img/well-architected/diagram-authenticate-tls.png#light-theme-only)
+
+</ImageConfig>
+
+<ImageConfig hideBorder>
+
+![Authenticate applications with TLS certificates through Vault Agent](/img/well-architected/diagram-authenticate-tls-dark.png#dark-theme-only)
+
+</ImageConfig>
+
+Vault's TLS certificate auth method allows authentication using SSL/TLS client certificates which are either signed by a CA or self-signed. Vault Agent is a client-side daemon that makes requests to Vault on behalf of a client application. This includes the authentication to Vault with TLS certificates through the Vault Auto-Auth cert method.
+
+You can also use Vault Agent to template secrets with the Consul Template syntax, and render the secrets to files where the client application can use them. Vault Agent also has a process supervision feature to send commands or signals to services to reload or restart them after templated credential rotation. 
+
+HashiCorp resources:
+
+- [Vault Agent and Vault Proxy quick start](/vault/tutorials/vault-agent/agent-quick-start)
+- [Vault Agent Templates](/vault/tutorials/vault-agent/agent-templates)
+- [Vault Auto-Auth cert method](/vault/docs/agent-and-proxy/autoauth/methods/cert)
+- [TLS certificates auth method](/vault/docs/auth/cert)
+- [TLS certificate auth method (API)](/vault/api-docs/auth/cert)
+- [Response wrapping](/vault/docs/concepts/response-wrapping)
+	
+### Authenticate application workloads with Consul
+
+You can implement a zero trust security model by authenticating and authorizing all service to service communications with Consul to mitigate unauthorized access and lateral movement within your network environments. Consul also enables fine grained access control through ACL policies, which you can use to limit access to specific API endpoints and service mesh features.
+
+<ImageConfig hideBorder caption="Authenticate services with Consul and TLS certificates">
+
+![Authenticate services with Consul and TLS certificates](/img/well-architected/diagram-secure-cluster-tls-consul.png#light-theme-only)
+
+</ImageConfig>
+
+<ImageConfig hideBorder>
+
+![Authenticate services with Consul and TLS certificates](/img/well-architected/diagram-secure-cluster-tls-consul-dark.png#dark-theme-only)
+
+</ImageConfig>
+
+When you authenticate application workloads with Consul, you eliminate hard coded credentials and addresses. You also improve compliance through audit trails, monitoring, and centralized policy management. You can also define your policies as code, version control them, and quickly update policies as requirements change.
+
+You can authenticate your application workloads on Consul with the following methods:
+
+- **ACL tokens:** Consul uses ACL tokens as its primary method of authentication. ACL tokens can take the form of static tokens or dynamic tokens backed by an auth method.
```

Comment:
from my understanding, acl tokens is used more to authenticate to the consul cluster rather than the services. this is how you connect to consul to send it commands

```suggestion
```